sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -85,7 +85,10 @@ public OfferEvaluator(StateStore stateStore, OfferRequirementProvider offerRequi
     public List<OfferEvaluationStage> getEvaluationPipeline(OfferRequirement offerRequirement) {
         List<OfferEvaluationStage> evaluationPipeline = new ArrayList<>();
 
+        // OfferRequirement will not include placement rules,
+        // so it skips placement rules if it is a restart or transient failure
```

Comment:
This is commented elsewhere, and that functionality isn't implemented here.  Changes to this file should be removed.